app/client/components/nav/dropdownNav.tsx

```diff
@@ -110,6 +110,10 @@ export const DropdownNav = () => {
     };
   });
 
+  const isHelpCentre = window.location.pathname.startsWith("/help-centre")
```

Comment:
You don't need the `? true : false` here as startsWith already returns a boolean.

Reply:
Thanks Paul, I missed that and will update it now 👍